src/main/scala/org/globalforestwatch/summarystats/firealerts/FireAlertsAnalysis.scala

```diff
@@ -0,0 +1,105 @@
+package org.globalforestwatch.summarystats.firealerts
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+import geotrellis.spark.SpatialKey
+import geotrellis.spark.tiling.LayoutDefinition
+import geotrellis.vector.{Feature, Geometry, Point}
+import org.apache.spark.{HashPartitioner, Partitioner}
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.{Row, SparkSession}
+import org.globalforestwatch.features.{FeatureId, FireAlertFeatureId}
+import org.globalforestwatch.summarystats.firealerts.FireAlertsRDD.SUMMARY
+import org.globalforestwatch.util.Util._
+
+import scala.reflect.ClassTag
+
+object FireAlertsAnalysis {
+  def apply(featureRDD: RDD[Feature[Geometry, FeatureId]],
+            featureType: String,
+            part: HashPartitioner,
+            spark: SparkSession,
+            kwargs: Map[String, Any]): Unit = {
+
+    import spark.implicits._
+
+    val fireAlertFeatureRDD = getFireAlertFeatureRDD(featureRDD, FireAlertsGrid.blockTileGrid, part, spark, kwargs)
+
+    val summaryRDD: RDD[(FireAlertFeatureId, FireAlertsSummary)] =
+      FireAlertsRDD(fireAlertFeatureRDD, FireAlertsGrid.blockTileGrid, part, kwargs)
+
+    val summaryDF =
+      FireAlertsDFFactory(featureType, summaryRDD, spark).getDataFrame
+
+    summaryDF.repartition(partitionExprs = $"id")
+
+    val runOutputUrl: String = getAnyMapValue[String](kwargs, "outputUrl") +
+      "/fireAlerts_" + DateTimeFormatter
+      .ofPattern("yyyyMMdd_HHmm")
+      .format(LocalDateTime.now)
+
+    FireAlertsExport.export(
+      featureType,
+      summaryDF,
+      runOutputUrl,
+      kwargs
+    )
+  }
+
+
+  def getFireAlertFeatureRDD(featureRDD: RDD[Feature[Geometry, FeatureId]],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=5008474)